lib/config/migration.ts

```diff
@@ -3,6 +3,7 @@ import is from '@sindresorhus/is';
 import { dequal } from 'dequal';
 import { logger } from '../logger';
 import { clone } from '../util/clone';
+import { fixShortHours } from '../util/date';
```

Comment:
@viceice are you sure this shouldn't remain part of migration?

Reply:
sure, but that's maybe a separate PR?

Reply:
@RahulGautamSingh please skip this one for now, it should be a separate issue (to move it to migration)